modules/codegen/src/main/scala/com/twilio/guardrail/terms/SwaggerTerm.scala

```diff
@@ -157,8 +158,18 @@ case class RouteMeta(path: String, method: HttpMethod, operation: Operation) {
       })
 }
 
+sealed trait SecurityScheme {
+  def typeName: Option[String]
```

Comment:
Why are these `String`?  Is this to avoid `L#Type`?

Reply:
We could do `L#Type`, but unfortunately `SwaggerGenerator` doesn't have an instance of `ScalaTerms`.

Reply:
Forgot about that point. We could add it, as there's no penalty, but we'll see what the consumer looks like

Reply:
I have a consumer in a branch that doesn't yet work... it ends up feeding the string to something that returns the language-equivalent of `L#Type`, so this _would_ be useful.

Reply:
On further reflection, I can't actually see a good way to call `parseType` given what we end up with here. The only alternative is something like a `case class SecuritySchemeConstructor[L <: LA](tpe: String, builder: L# Type => SecurityScheme)`, but it's clearly working around the paradigms, not working with them.

Thinking through how we resolve this practice in other places, consumers of `extractSecuritySchemes` could be rewritten to:
```scala
      securitySchemes <- Option(swagger.getComponents)
        .flatMap(components => Option(components.getSecuritySchemes))
        .map(_.asScala.toMap)
        .getOrElse(Free.pure[F, Map[String, SwSecurityScheme]](Map.empty))
        .toList
        .traverse { case (schemeName, scheme) =>
          // Decompose processor behaviour into distinct chunks with explicit extractors instead of a monolith that owns all responsibility
        }
```
... but I wonder if this is ceremony for ceremony's sake. What do you think?

Reply:
I actually started going down that path in the first place, but then found that I had to copy/paste the same code into the client generator, server generator, and one of the test runners, so I opted to do it this way.

Alternatively, I could stash this code in `SwaggerUtil` and just call it from those three places.  That way I think we'd be able to do `parseType()` there as well, right?

Reply:
Yeah; a lot of `SwaggerUtil` is becoming just `Free` utility functions, so this would be reasonable. The motivator here is to not accidentally hard couple exactly which type parser we're using, keeping everything overridable by interpreters. Thanks for considering this approach!